api/add.go

```diff
@@ -0,0 +1,32 @@
+package api
+
+// AddOption is the options for the Add API.
+type AddOption struct {
+	Url   string `json:"url,omitempty"`
+	Title string `json:"title,omitempty"`
+	Tags  string `json:"tags,omitempty"`
+}
+
+type addAPIOptionWithAuth struct {
+	*AddOption
+	authInfo
+}
+
+type AddResult struct{}
+
+// Only returns an error status, since adding an article doesn't have
```

Comment:
Can you start this doc with "Add ..." as [golint](https://github.com/golang/lint) complains? (Though other part of this project does not satisfy golint :stuck_out_tongue_winking_eye: )